climt/_components/rrtmg/sw/component.py

```diff
@@ -196,7 +200,10 @@ def __init__(
             ignore_day_of_year=False,
             facular_sunspot_amplitude=None,
             solar_variability_by_band=None,
-            aerosol_type='no_aerosol', **kwargs):
+            aerosol_type='no_aerosol',
```

Comment:
You should add a description of what these additional keyword arguments do to the docstring of this method.